l10n_eu_product_adr_report/report/product_adr_report.xml

```diff
@@ -1,91 +1,90 @@
 <odoo>
     <data>
-
         <report
             id="product_adr_report"
             model="stock.picking"
             string="ADR product"
             report_type="qweb-pdf"
             name="l10n_eu_product_adr_report.product_adr_report_document"
-            file="l10n_eu_product_adr_report.product_adr_report"
+            file="l10n_eu_product_adr_report.product_adr_report_document"
             attachment_use="False"
         />
 
         <template id="product_adr_report_document">
-
-            <div class="page">
```

Comment:
You shouldn't remove this div. Try  to put it after  <t t-call="web.external_layout">